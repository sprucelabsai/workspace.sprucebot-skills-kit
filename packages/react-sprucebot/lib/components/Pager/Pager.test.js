'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _Pager = require('./Pager');

var _Pager2 = _interopRequireDefault(_Pager);

var _enzyme = require('enzyme');

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

describe('Pager tests', function () {
	it('Should match the snapshot', function () {
		var renderedComponent = (0, _enzyme.shallow)(_react2.default.createElement(_Pager2.default, null));

		expect(renderedComponent).toMatchSnapshot();
	});
});