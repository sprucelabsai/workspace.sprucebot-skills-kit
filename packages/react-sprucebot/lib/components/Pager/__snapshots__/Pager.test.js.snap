// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<ul
  className="pager"
>
  <li
    className="first disabled"
    onClick={[Function]}
  >
    <a>
      First
    </a>
  </li>
  <li
    className="back disabled"
    onClick={[Function]}
  >
    <a>
      Back
    </a>
  </li>
  <li
    className="current"
  >
    <a>
      1 of 1
    </a>
  </li>
  <li
    className="next disabled"
    onClick={[Function]}
  >
    <a>
      Next
    </a>
  </li>
  <li
    className="last disabled"
    onClick={[Function]}
  >
    <a>
      Last
    </a>
  </li>
</ul>
`;
